services/backend/util/keycloak.js

```diff
@@ -8,10 +8,10 @@ const keycloak = new Keycloak({ store: memoryStore });
 require("dotenv").config();
 
 const sessionInstance = session({
-  secret: process.env.KEYCLOAK_SECRET,
+  secret: process.env.DEV_KEYCLOAK_SECRET,
```

Comment:
This is not how it should be done

Reply:
I'm not done yet! My PR is still not ready for review